.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,62 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+  schedule:
+    - cron: '0 18 * * 5'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # Override automatic language detection by changing the below list
+        # Supported options are ['csharp', 'cpp', 'go', 'java', 'javascript', 'python']
+        language: ['javascript']
```

Comment:
Can we also add typescript to it?

Since out microservices are written in typescript.

Reply:
For the current Beta release, only JavaScript is supported. I'll add Typescript once it becomes available for code scanning.

Reply:
Later I am planning to look into https://github.com/SonarSource/SonarJS